src/ast/nodes/Identifier.ts

```diff
@@ -149,7 +149,7 @@ export default class Identifier extends NodeBase {
 						break;
 					}
 					code.overwrite(expression.start, expression.end,
-						`(exports(${this.variable.exportName}, ${op}), ${name}${expression.operator})`);
+						`(exports('${this.variable.exportName}', ${op}), ${name}${expression.operator})`);
```

Comment:
Good catch, but do you think we might reflect this in a test somewhere?

Reply:
Let me finally get around to adding system format expectations for the "form" folder in general, which would catch all these cases. Will aim to do this as a separate PR though. Otherwise I'm just going to be continuing to pollute the chunking-form tests to try and cover everything further.

Reply:
That would be awesome!